tests/test_modeling_tf_bert.py

```diff
@@ -328,3 +328,27 @@ def test_custom_load_tf_weights(self):
         self.assertEqual(sorted(output_loading_info["unexpected_keys"]), ["mlm___cls", "nsp___cls"])
         for layer in output_loading_info["missing_keys"]:
             self.assertTrue(layer.split("_")[0] in ["dropout", "classifier"])
+
+
+class TFBertModelIntegrationTest(unittest.TestCase):
+    @slow
+    def test_inference_masked_lm(self):
+        model = TFBertForPreTraining.from_pretrained("lysandre/tiny-bert-random", use_cdn=False)
```

Comment:
Will remove the `use_cdn=False` tomorrow once the CDN caching has caught up.

```suggestion
        model = TFBertForPreTraining.from_pretrained("lysandre/tiny-bert-random")
```